src/sources/static_metrics.rs

```diff
@@ -58,14 +58,21 @@ impl Default for StaticMetricsConfig {
 
 /// Tag configuration for the `internal_metrics` source.
 #[configurable_component]
-#[derive(Clone, Debug, Default)]
+#[derive(Clone, Debug)]
 #[serde(deny_unknown_fields, default)]
 pub struct StaticMetricConfig {
     /// Name of the static metric
     pub name: String,
 
     /// "Observed" value of the static metric
-    pub value: f64,
+    #[configurable(derived)]
```

Comment:
Is this attribute necessary? (`#[configurable(derived)]`) I think it does nothing here 🤔  (also in the `kind` field)